web/themes/custom/pineapple/templates/paragraphs/paragraph--grid-item.html.twig

```diff
@@ -0,0 +1,79 @@
+ *   - paragraph.id(): The paragraph ID.
+ *   - paragraph.bundle(): The type of the paragraph, for example, "image" or "text".
+ *   - paragraph.getOwnerId(): The user ID of the paragraph author.
+ *   See Drupal\paragraphs\Entity\Paragraph for a full list of public properties
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+  set classes = [
+    'paragraph',
+    'paragraph--type--' ~ paragraph.bundle|clean_class,
+    view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+    not paragraph.isPublished() ? 'paragraph--unpublished'
+  ]
+%}
+{% set image_paragraph = content.field_p_gi_image|field_value|first['#paragraph'] %}
+{% set alt_text = image_paragraph.field_p_ip_image.entity.field_media_image.alt %}
+{% if paragraph.field_p_gi_layout.value == 'Wide Teaser'%}
+  {% set grid_image = bamboo_render_image_style(image_paragraph.field_p_ip_image.entity.field_media_image.entity.uri.value, 'wide_teaser_660_x_330') %}
+{% else %}
+  {% set grid_image = bamboo_render_image_style(image_paragraph.field_p_ip_image.entity.field_media_image.entity.uri.value, 'default_teaser_330_x_330') %}
+{% endif %}
+
+{% set image_attributes = image_attributes|default(create_attribute()).addClass([
+  content.field_p_gi_link|field_value ? 'linked'
+]).setAttribute('src', grid_image).setAttribute('alt', alt_text) %}
+
+{% if content.field_p_gi_link|field_value %}
+  {% set url_prefix %}
+    <a href="{{ content.field_p_gi_link|field_value|first['#url'] }}" class="image-link">
+  {% endset %}
+  {% set url_suffix %}</a>{% endset %}
+{% endif %}
+
+<div{{ attributes.addClass(classes) }}>
+  <div class="grid-image-container">
```

Comment:
Either remove the div on line 68 or move the class on line 69 into the classes array and remove that div. My point is you don't need both divs.

Reply:
done, thanks!